### ISSUE TITLE
parser fails on contains

### ISSUE THREAD

**Issue by ouven** [markglenn/liquex#12]
Hi,

first I want to thank you for this library! Its exzellent extendibility made it to our choice :)

I don't know how to reproduce this in an isolated test but it happens to be, that in some constellations, the parser fails, on the operation "contains".

I found this piece of code at https://github.com/markglenn/liquex/blob/ba9896e24d1177f5b93d97bcb52d751fba9f1a71/lib/liquex/parser/tag/control_flow.ex#L19

```
 def boolean_expression(combinator \\ empty()) do
    operator =
      choice([
        string("=="),
        string("!="),
        string(">="),
        string("<="),
        string(">"),
        string("<"),
        string("contains")
      ])
      |> map({String, :to_existing_atom, []})

```

Am I wrong, that this doesn't need to a `to_existing_atom`, as the choice is already a finit set?

## example
```
{% assign lang = "de,fr" | split: "," %}
{% if lang contains "de" %}
  DE
{% else %}
  OTHER
{% endif %}
```

## Failure

Sorry, I had to x-out parts of the stacktrace. But I guess the point is clear :)
```
     *****_test.exs:28
     ** (ArgumentError) errors were found at the given arguments:
     
       * 1st argument: invalid UTF8 encoding
     
     stacktrace:
       :erlang.binary_to_existing_atom("contains", :utf8)
       ***** anonymous fn/1 in *****.document__6897/6
       (elixir 1.12.1) lib/enum.ex:1553: Enum."-map/2-lists^map/1-0-"/2
       *****.document__6897/6
       *****.parse__0/6
       *****.parse/2
       (liquex 0.6.0) lib/liquex.ex:178: Liquex.parse/2

```

I prepared a pr: #13

**Comment by markglenn** [comment]
Thank you for this.  You're right that `:contains` likely doesn't exist in the existing atom memory space unlike the rest of the operators.  You will also have trouble reproducing this because the second you add `:contains` into your code, this will start working.  Using `String.to_atom/1` is the right choice.

I merged your PR #13 and I'll push a new release later this week.  Thank you for the help.

**Comment by markglenn** [comment]
v[0.6.1](https://hex.pm/packages/liquex/0.6.1) has been published.
